README.md

```diff
@@ -72,11 +72,22 @@ end
 
 To install the latest release as a gem:
 
-`gem install rack-test`
+```
+gem install rack-test
+```
 
 Or via Bundler:
 
-`gem "rack-test", require: "rack/test"`
+```
+gem 'rack-test', require: 'rack/test'
+```
+
+Or to install unreleased version via Bundler:
+
+```
+gem 'rack-test', github: 'rack-test', branch: 'master'
+gem 'rack-test', github: 'rack-test', ref: 'a5aa1e5'
```

Comment:
I think we could skip the second line. Most users will probably not go with a specific ref anyway, but rather a named _branch_.